app/components/groupe_gestionnaire/groupe_gestionnaire_list_commentaires/commentaire_component.rb

```diff
@@ -8,9 +8,9 @@ def initialize(groupe_gestionnaire:, commentaire:)
 
   def email
     if @commentaire.sender == current_gestionnaire
-      "Messages avec le groupe gestionnaire parent (#{@groupe_gestionnaire.parent.name})"
+      "Messages avec le groupe gestionnaire parent"
     else
-      @commentaire.sender_email
+      @commentaire.sender_email + (@commentaire.groupe_gestionnaire_id == @groupe_gestionnaire.parent_id || @commentaire.groupe_gestionnaire_id == @groupe_gestionnaire.id ? '' : " (#{@commentaire.groupe_gestionnaire.name})")
```

Comment:
Proposition un peu plus facile à suivre : 

```suggestion
if @commentaire.sender == current_gestionnaire
  "Messages avec le groupe gestionnaire parent"
elsif @commentaire.groupe_gestionnaire_id.in?([@groupe_gestionnaire.parent_id, @groupe_gestionnaire.id])
  @commentaire.sender_email
else
  "#{@commentaire.sender_email} (#{@commentaire.groupe_gestionnaire.name})"
end
```

Reply:
done